### DISCUSSION TITLE
How to get rid of undo / redo buttons?

### DISCUSSION THREAD

**Discussion by printmaps** [devemux86/cruiser#655] [Q&A] [answered]
How to get rid of the undo / redo buttons? An application restart does this. But how to achieve this without restart?

![Screenshot_20241026_092025_Cruiser](https://github.com/user-attachments/assets/20fdbc01-f4bd-4944-a86c-443e897b86c7)

**Comment by devemux86** [comment]
You can enable the auto hide of the buttons in "Settings | Application | Screen | Auto hide".

<img src="https://github.com/user-attachments/assets/8ce128de-14c7-480d-8d7d-4be9934b39c2" width="50%"/>

**Comment by printmaps** [comment]
To hide the buttons is not what I meant. I purposely want to get rid of the undo/redo functionality and/or the buttons at a specific point. E. g.: After 30 minutes I don't know anymore what I get when I press these buttons.

**Comment by devemux86** [comment]
Previously we had an "X" button in the middle to close the undo/redo buttons.
We can restore it, but if we close them, they can only reappear when we do a new route calculation.

**Reply by devemux86** [reply]
> Previously we had an "X" button in the middle to close the undo/redo buttons.

Available in Cruiser 5.0.1-b13.

**Comment by devemux86** [comment, marked as answer]
- Available in https://github.com/devemux86/cruiser/discussions/641
